src/main/java/com/github/markusbernhardt/seleniumlibrary/keywords/BrowserManagement.java

```diff
@@ -184,13 +185,18 @@ public void closeBrowser() {
             "\r\n" + 
             "If the provided configuration options are not enough, it is possible to use `Create Webdriver` to customize browser initialization even more.")
     @ArgumentNames({ "url", "browserName=firefox", "alias=None", "remoteUrl=None", "desiredCapabilities=None",
-            "browserOptions=None" })
-    public String openBrowser(String url, String... args) throws Throwable {
+            "browserOptions=None", "webDriverManager=None" })
+    public String openBrowser(String url, String... args) {
         String browserName = robot.getParamsValue(args, 0, "firefox");
         String alias = robot.getParamsValue(args, 1, "None");
         String remoteUrl = robot.getParamsValue(args, 2, "None");
         String desiredCapabilities = robot.getParamsValue(args, 3, "None");
         String browserOptions = robot.getParamsValue(args, 4, "None");
+        String webDriverManager = robot.getParamsValue(args, 5, "None");
+
+        if (!webDriverManager.equals("None")) {
```

Comment:
It would be cleaner, if Robot -class itself would take boolean as a default, and use Boolean value also here as there's no usage of String value in this case.

Reply:
Done